plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-checkout-calculate-tax.spec.js

```diff
@@ -267,13 +271,13 @@ test.describe.serial(
 						.locator( '#billing_state' )
 						.selectOption( customer.billing.us.state );
 					await page
-						.getByRole( 'textbox', { name: 'ZIP Code *' } )
+						.getByRole( 'textbox', { name: 'ZIP Code required' } )
 						.type( customer.billing.us.zip );
 					await page
-						.getByLabel( 'Phone *' )
+						.getByLabel( 'Phone * required' )
```

Comment:
The `getByLabel` method includes the asterisk even if it has the `aria-hidden="true"` attribute. That's why it hasn't been removed in this case.